tactics/induction.ml

```diff
@@ -0,0 +1,1571 @@
+(************************************************************************)
+(*         *   The Coq Proof Assistant / The Coq Development Team       *)
+(*  v      *         Copyright INRIA, CNRS and contributors             *)
+(* <O___,, * (see version control and CREDITS file for authors & dates) *)
+(*   \VV/  **************************************************************)
+(*    //   *    This file is distributed under the terms of the         *)
+(*         *     GNU Lesser General Public License Version 2.1          *)
+(*         *     (see LICENSE file for the text of the license)         *)
+(************************************************************************)
+
+module CVars = Vars
+
+open Pp
```

Comment:
While we are here, is it possible to make this list of opens shorter?

Reply:
Even if we did that, I'd rather have it in another PR so as to keep as many lines recognized to have been moved by git as possible.

Reply:
Sure thing.